### ISSUE TITLE
As an employee I shouldn't see any buttons if the Incident has been resolved

### ISSUE THREAD

**Issue by j0nscalet** [j0nscalet/QualityKitchen.Mobile#67]
**Problem**
For the sake of this issue let's define the comment area as: comment box, comment, fix, and resolve buttons as well as the animation on the comment textbox that makes room for the keyboard.

Here's a screenshot for some context:
![image](https://cloud.githubusercontent.com/assets/1328795/25102856/0dde6dbc-236f-11e7-958e-d897134deea7.png)

Currently after an incident is resolved the comment area is still enabled and visible, it shouldn't be. Instead:
- The comment box animation should be turned off
- The 'Comment', 'Fix', and 'Resolve' should be disabled
    - If you are an employee the comment area shouldn't be visible at all
    - If you're a manager the comment area should be disabled

**Solution**
[Describe potential solutions to the issue. For Design issues you should be writing code in this section]

**Comment by j0nscalet** [comment]
@ershovd I need some time to re-work this issue a bit. There's some dependent bugs that I've uncovered as well that affect this too as well as some questions I need to answer first.

**Comment by j0nscalet** [comment]
@ershovd Ok I think we're good. Like we discussed on this one, can you make sure that the 'Resolve' and 'Post' (Comment) buttons are disabled? That should suffice for now.

**Comment by ershovd** [comment]
@j0nscalet ok, I will make that the comment area is not visible (if you an employee) and that the area is shown but disable if you are a manager.
Do you plan to enable or turn-on the area for the manager? Otherwise, not sure why he need to see this area, but not be able to make any actions with it?

**Comment by j0nscalet** [comment]
Great, for now let's just disable the buttons for both roles. In the future managers should be able to re-open issues. Initially it would make sure to hide the area for employees but he/she may might to re-open it as well if they notice that the incident is not resolved.

For example, there's an incident with an 'Oven Malfunction - it's not working', the team works together and it's resolved by a manager. Now, three days later the issue starts to occur again. In order to keep the number of incident down we may decide to allow the employee to re-open an issue as well. Because of this I think it's best to simply disable the buttons for now.

Can you tweak the UI for the buttons to make sure that they actually show disabled? I think because we are using implicit styles a button has a tendency to show in it's normal state instead of a disabled one.

Thanks!

**Comment by j0nscalet** [comment]
@ershovd No need to be concerned with this issue for now. @danielqualitykitchen and I need to figure out the functionality so everyone is on the same page. 

I'll circle back and finish the review of your pull request once that is done.

**Comment by j0nscalet** [comment]
@danielqualitykitchen standby for my comments on this. 

I want to dig up the related issues from the last feedback cycle to make sure we have a clear issue description. We just need to iron out functionality for this at a stop-gap until a manager can re-open an incident.

**Comment by j0nscalet** [comment]
This is now a duplicate of #113.
